addons/mail/static/src/core/common/composer.xml

```diff
@@ -235,4 +235,12 @@
             <t t-esc="option.label"/>
         </em>
     </t>
+    <t t-name="mail.Composer.suggestionEmoji">
+        <strong class="px-2 py-1 align-self-center flex-shrink-1 text-truncate">
+            <t t-esc="option.emoji.codepoints"/>
+        </strong>
+        <em class="text-600 text-truncate align-self-center" style="flex-basis: 20%;">
+            <t t-esc="option.emoji.shortcodes.join(', ')"/>
+        </em>
+    </t>
```

Comment:
`flex-basis: 20%` is workaround for canned response description to not truncate as much as the canned code.
This is a non-issue with emoji: the codepoints is never truncated, and this on is simply truncated.